src/pages/office/components/language-preference/__tests__/language-preferences.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { LanguagePreferencesComponent } from '../language-preferences';
+import { IonicModule } from 'ionic-angular';
+import { AppModule } from '../../../../../app/app.module';
+import { FormGroup } from '@angular/forms';
+
+describe('LanguagePreferencesComponent', () => {
+  let fixture: ComponentFixture<LanguagePreferencesComponent>;
+  let component: LanguagePreferencesComponent;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        LanguagePreferencesComponent,
+      ],
+      imports: [
+        IonicModule,
+        AppModule,
+      ],
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(LanguagePreferencesComponent);
+        component = fixture.componentInstance;
+        component.formGroup = new FormGroup({});
+      });
+  }));
+
+  describe('Class', () => {
+    it('should compile', () => {
```

Comment:
Don't bother with this test if you've got other tests that actually do something to the component.